### ISSUE TITLE
Research: How does assignments work together with diagrams

### ISSUE THREAD

**Issue by b21phiro** [HGustavs/LenaSYS#13974]
## Description
We need to know how the assignments and the diagrams work together when a new assignment is made. 

## Your job
Together with another person, do some research about how this works, and write a quick report about it.

**Comment by b21willu** [comment]
I have tested adding new assignments with diagrams.
In issue #13872, Henrik wrote, "As I remember it, an assignment can give a starting diagram that the users can modify as part of that assignment, e.g., repair this diagram." Based on what I have tested, this is not possible when creating a new assignment. It is possible to open example diagrams, but this happens manually in the diagram once the assignment has already been created.
In the same issue, it was also mentioned, "As I recall, there should also be another mode that has the original given diagram section write protected so that the user cannot modify that given starting diagram." This also does not seem to work, as far as I can tell. All assignments with diagrams appear the same, and there are no specific options to change the assignment. Essentially, it is an empty diagram with an instruction list on the left containing pre-written instructions for the student to follow. 
Therefore, it seems that there are no different options for creating new assignments with diagrams. So far, there is only one task, and to change the instructions text someone would have to go in and modify the code. So, to summarize there doesn't seem to be any way to modify to make a new task to the student when creating a new assignmen with diagrams.
